cprover_bindings/src/machine_model.rs

```diff
@@ -8,27 +8,27 @@ pub struct MachineModel {
     /// Minimum architectural alignment, in bytes
     /// The name of the architecture
     /// Width of a pointer, in bits
-    alignment: u64,
-    architecture: String,
-    bool_width: u64,
-    char_is_unsigned: bool,
-    char_width: u64,
-    double_width: u64,
-    float_width: u64,
-    int_width: u64,
-    is_big_endian: bool,
-    long_double_width: u64,
-    long_int_width: u64,
-    long_long_int_width: u64,
-    memory_operand_size: u64,
-    null_is_zero: bool,
-    pointer_width: u64,
-    rounding_mode: RoundingMode,
-    short_int_width: u64,
-    single_width: u64,
-    wchar_t_is_unsigned: bool,
-    wchar_t_width: u64,
-    word_size: u64,
+    pub alignment: u64,
+    pub architecture: String,
```

Comment:
Not that it really matters for memory usage, but should we used `InternedString` here?

Reply:
_Probably_ not, I don't think we see duplicates of this structure in memory.